lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -795,7 +816,9 @@ private function columnToArray(SimpleXMLElement $fieldMapping)
      *
      * @param SimpleXMLElement $cacheMapping
      *
-     * @return array
+     * @return mixed[]
+     *
+     * @psalm-return array{usage: mixed, region: null|string}
```

Comment:
If I'm not mistaken CS enforces `null` to be the last type.

```suggestion
     * @psalm-return array{usage: mixed, region: string|null}
```

Reply:
You are correct :) , it just doesn't enforce it here, let's change that.